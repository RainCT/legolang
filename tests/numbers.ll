int i
float f

f = 1.5e4
print f

i = f
print i

i = 1.5e4
print i

print ""

// ---

f = 34.240e-3
//print f

f = -34.240e-3
//print f

f = -34.240e2
print f

print ""

// ---

//f = 1e6
//i = 1e6
//print f + " " + i
