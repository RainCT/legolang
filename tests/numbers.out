15000.0
15000
15000

-3424.0

